tests/unit/logic/multisignature.spec.js

```diff
@@ -217,9 +183,10 @@ describe("modules/multisignature", function() {
 			verifySignature = sinon
 				.stub(library.logic.transaction, "verifySignature")
 				.returns(true);
-			Buffer = Multisignature.__get__("Buffer");
-			from = sinon.stub(Buffer, "from").returns(fullArray);
-			Multisignature.__set__("Buffer", Buffer);
+      var returnBuff = new Buffer('00000000000000000000000000000000');
+      Buff = Multisignature.__get__("Buffer");
```

Comment:
Indentation :)

Reply:
Also I don't like the 'Invalid public key in mutlisignature keysgroup' test. You do `from.throws` to make it fail. but you don't test the length branch and also i think rewiring Buffer.from is not the way to go. please remove that stub and pass data instead that should pass/fail.

Reply:
👍

Reply:
Indentation fixed, but I think I still need to stub this buffer. Otherwise I can't really reproduce the throw, due previous checks ( logic/multisignature.js:127 and test: tests/unit/logic/multisignature.spec.js:379)

Reply:
https://github.com/RiseVision/rise-node/blob/development/logic/multisignature.js#L127 ?

Reply:
https://github.com/RiseVision/rise-node/blob/6c7c181acff82da38873cb013075faef82597aa2/tests/unit/logic/multisignature.spec.js#L379 ?

Reply:
Correct, that will prevent to pass a wrong parameter to the Buffer.from()